src/layout/components/page-top.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { Link } from 'react-router';
 import {SearchBar} from 'src/layout/components/search-bar';
+import moment from 'moment';
+import noop from 'lodash';
```

Comment:
`import {noop}` otherwise it aliases the full lodash library as the variable noop.